nservicebus/operations/metrics.md

```diff
@@ -71,4 +71,29 @@ These statistics encompass a number of different metrics, including:
 - Number of message processing failures
 - Number of messages successfully processed
 
+### Queue length
 
+This metric tracks estimated number of messages in the input queue of an endpoint.
+
+A _link_ is a communication channel between a sender of the message and its receiver. Each link is uniquely identified by some combination of destination address, message assembly, and the [host identifier](/nservicebus/hosting/override-hostid.md#host-identifier) of the sender. The exact composition of link identifiers depends on the transport properties and type of message being sent.
+
+Each sender maintains a monotonic counter of messages sent over each of its outgoing links and transmits the value of this counter to the receiver in a message header. The receiver tracks the counter value for the last message received over each link. This allows both communicating endpoints to track how many messages were sent and received over each link.
+
+ServiceControl collects these metrics for all links and calculates the estimates length of the input queue for each receiver based on how many messages were sent in total over all incoming links and how many of those messages have already been received.
+
+#### Example
+
+Consider the following values reported to ServiceControl:
+
+| Link ID                        | Max sent counter | Max received counter | Messages in queue from this link |
+|--------------------------------|:----------------:|:--------------------:|:--------------------------------:|
+| `Sales-client1.example.com`    | 10               | 6                    | 4                                |
+| `Sales-client2.example.com`    | 5                | 1                    | 4                                |
+| `Shipping-client1.example.com` | 1                | 1                    | 0                                |
+
+Based on the data above, ServiceControl can estimate the following values of queue length for `Sales` and `Shipping` endpoints:
```

Comment:
> Based on the data above

I think it may be worth explicitly stating that `Sales` is scaled out over the two `client1` and `client2` machines. That being said, the word `client` doesn't sound so good. I'd suggest `server` instead. The same would go for `Shipping-client1`